definitions/infra-container/definition.yml

```diff
@@ -50,6 +50,21 @@ synthesis:
         entityTagName: k8s.clusterName
       k8s.namespace.name:
         entityTagName: k8s.namespaceName
+  - identifier: container.id
+    name: container.name
+    encodeIdentifierInGUID: false
+    # This is a temporary condition to restrict synthesis for a particular set of telemetry data.
+    conditions:
+      - attribute: nr.entity_type
+        value: container
+    tags:
```

Comment:
Do we want to add one for the container state (it's present in the other rules)

Reply:
unfortunately this information doesn't come in the telemetry